docs/install.rst

```diff
@@ -494,7 +496,7 @@ build the documentation.
 
 If conda is being used then the other packages can be installed with::
 
-  conda install -c conda-forge pandoc graphviz
+  conda install -c conda-forge pandoc graphviz arviz
```

Comment:
Technically we should not need `arviz` since it should have been installed with `pip install .[doc]`, but I don't think it harms us to keep it here.

Reply:
you are right.